kin-devplatform-sdk/src/main/java/kin/devplatform/ConfigurationImpl.java

```diff
@@ -120,6 +121,10 @@ public Response intercept(Chain chain) throws IOException {
 		return defaultApiClient;
 	}
 
+	private boolean shouldSkipAuthentication(Request originalRequest, String path) {
+		return path.equals(USERS_PATH) && originalRequest.method().equals(POST) || path.contains(KIN_VERSION_PATH);
```

Comment:
better to add parenthesis to make it clear, also you use contains while USERS_PATH uses equals